frontend/src/ProfilePage/FormPages/PageOne.jsx

```diff
@@ -1,119 +1,148 @@
-import './PageOne.css'
-import Select from 'react-select'
+import './PageOne.css';
+import Select from 'react-select';
 import makeAnimated from 'react-select/animated';
 import { populateAccount } from '../../../HelperFuncs/utils';
 import { useState, useEffect } from 'react';
 import { useParams } from 'react-router-dom';
 
-function PageOne( {nextPage} ) {
-
-    const animated = makeAnimated();
+function PageOne({ nextPage }) {
+    const animatedComponents = makeAnimated();
+    const { userId } = useParams();
 
     /* 
     because the form is divided into two divs, accessing the form values is more
     nuanced requiring the use of this state object to track values
     */
     const [formData, setFormData] = useState({
-        city : '' , 
-        roommates : 0,
-        children : 0, 
-        salary : '',
+        city: '',
+        roommates: 0,
+        children: 0,
+        salary: '',
         job: ''
-    })
-
-    const { userId } = useParams();
+    });
+
+    const [jobs, setJobs] = useState([]);
+    const [searchTerm, setSearchTerm] = useState('');
+
+
+    useEffect(() => {
+        if (searchTerm) {
+            const where = encodeURIComponent(JSON.stringify({
+                "title": {
+                    "$regex": searchTerm,
+                    "$options": "i"
+                }
+            }));
+            const headers = {
+                'X-Parse-Application-Id': import.meta.env.VITE_API_BLS_ID,
+                'X-Parse-REST-API-Key': import.meta.env.VITE_API_BLS_KEY
+            };
+            fetch(`https://parseapi.back4app.com/classes/Occupations_Job?limit=10&keys=title&where=${where}`, {
```

Comment:
Would it be worth pulling this API call out into a separate file, like you have been doing with the other API calls?

Reply:
Yes, good point, I'll clean that up now